sos/report/plugins/openvswitch.py

```diff
@@ -29,6 +29,13 @@ def setup(self):
             '/usr/local/var/log/openvswitch/',
         ]
 
+        check_dpdk = False
+        dpdk_enabled = self.collect_cmd_output(
+            "ovs-vsctl -t 5 get Open_vSwitch . other_config:dpdk-init")
+        if dpdk_enabled["status"] == 0 and \
+           dpdk_enabled["output"].startswith("true"):
```

Comment:
Isn't it easier to have:

    check_dpdk = (dpdk_enabled["status"] == 0 and \
                            dpdk_enabled["output"].startswith("true"))

? As the line 32 now is superfluous.

Reply:
Backslash escaping a newline is superfluous if you're inside parenthesis ;-)